examples/data_pipeline/transforms.py

```diff
@@ -6,19 +6,6 @@
 from torch import Tensor
 
 
-class TextClassificationPipeline(nn.Module):
```

Comment:
Is this removed because it wasn't being used?

Reply:
We moved them to the experimental.transforms file.